nexenta-fusion-installer.sh

```diff
@@ -0,0 +1,320 @@
+#!/bin/bash
+
+ips=($(ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*'))
+defaultFusionPath="$HOME/fusion"
+containerName="nexenta-fusion"
+# text formatting variables
+textBold=$(tput bold)
+textNormal=$(tput sgr0)
+textRed='\033[0;31m'
+textBlue='\033[0;36m'
+textYellow='\033[38;5;226m'
+textLightGray='\033[1;37m'
+textNc='\033[0m' # N
```

Comment:
Are you sure all this fancy stuff will work on any terminal?
Maybe somehow define is terminal supported these escapes and alternate assignment these variables to empty

Reply:
I can't be 100% sure while I can't test on every terminal, but all major terminals support this ( I tested). After all, if some terminal does not support this, it's not a problem, it's just text colors

Reply:
I know somehow there is a way to define can current session support colors or not.

Reply:
ok, let's do it if we have any problems with it